core/runtime/jsruntime.rs

```diff
@@ -1499,16 +1498,13 @@ impl JsRuntime {
       {
         // pass, will be polled again
       } else {
-        let scope = &mut self.handle_scope();
-        let messages = module_map.borrow().find_stalled_top_level_await(scope);
-        // We are gonna print only a single message to provide a nice formatting
-        // with source line of offending promise shown. Once user fixed it, then
-        // they will get another error message for the next promise (but this
-        // situation is gonna be very rare, if ever happening).
-        assert!(!messages.is_empty());
-        let msg = v8::Local::new(scope, messages[0].clone());
-        let js_error = JsError::from_v8_message(scope, msg);
-        return Poll::Ready(Err(js_error.into()));
+        let known_realms = self.inner.state.borrow().known_realms.clone();
```

Comment:
I'm not sure about this one -- this will be extremely expensive. Can we pass the slice into find_and_report_stalled_level_await_in_any_realm instead?